cmd/kubectl-gadget/utils/trace.go

```diff
@@ -36,11 +36,13 @@ import (
 	restclient "k8s.io/client-go/rest"
 
 	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
+	"github.com/kinvolk/inspektor-gadget/pkg/gadgets"
```

Comment:
I think this package should not be imported from `cmd/`. It is only used for `gadgets.ErrNoMatch.Error()` but I think we should avoid string comparisons of errors between the CLI and the gadgets (see my other comment).

Reply:
I understood your point. I thought also about using standard errors but it would not resolve what we are discussing in the other comment. Let's continue there.